assets/src/dashboard/components/cardGrid/index.js

```diff
@@ -37,7 +37,7 @@ const DashboardGrid = styled.div(
   grid-template-columns:
     repeat(auto-fill, ${columnWidth}px);
   grid-template-rows: minmax(${columnHeight}px, auto);
-  scroll-margin-top: 50vh;
+  scroll-margin-top: 30vh;
```

Comment:
50vh was a little excessive! 30vh still keeps the top of the grid visible on initial focus as space permits.